config/base.json

```diff
@@ -1,3 +1,9 @@
 {
-    "urls": {}
+    "urls": {},
+    "stripeKey": "pk_test_KPoCfZXu7mznb9uSCPZ2JpTD",
+    "btKey": "sandbox_r72q8jq6_9pnxkwm75f87sdc2",
```

Comment:
Could we rename this to `braintreeKey`? It's not obvious at first glance what `bt` stands for.